src/modm/board/samg55_xplained_pro/board.hpp

```diff
@@ -51,5 +66,12 @@ initialize()
 	Button::setInput();
 }
 
+inline void initializeUsbDevice()
```

Comment:
```suggestion
inline void initializeUsbFs()
```

The idea is to be able to compile the [`example/generic/usb` example](https://github.com/modm-io/modm/blob/develop/examples/generic/usb/main.cpp#L41), by only switching the BSP in the `project.xml`. Not entirely sure if there are more changes required though.